build/docker/intel-dsa-initcontainer.Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Copyright 2021 Intel Corporation. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:buster-slim AS builder
+
+RUN apt-get update && apt-get install -y \
+            build-essential autoconf automake autotools-dev libtool \
+            pkgconf asciidoc xmlto uuid-dev libjson-c-dev libkmod-dev \
+            libudev-dev libkeyutils-dev curl
+
+ENV ACCEL_CONFIG_VERSION v2.8
+ENV ACCEL_CONFIG_DOWNLOAD_URL https://github.com/intel/idxd-config/archive/accel-config-$ACCEL_CONFIG_VERSION.tar.gz
+ENV ACCEL_CONFIG_SHA256 c8aa8421179b71fb728806e5a621655fab24a862f9ce982f3119b92da416949f
+
+RUN curl -fsSL "$ACCEL_CONFIG_DOWNLOAD_URL" -o accel-config.tar.gz \
+    && echo "$ACCEL_CONFIG_SHA256 accel-config.tar.gz" | sha256sum -c - \
+    && tar -xzf accel-config.tar.gz \
+    && rm accel-config.tar.gz
+
+RUN cd idxd-config-accel-config-$ACCEL_CONFIG_VERSION && \
+    mkdir m4 && \
+    ./autogen.sh && \
+    ./configure CFLAGS='-g -O2' --prefix=/usr --sysconfdir=/etc --libdir=/usr/lib64 --disable-test --disable-docs && \
+    make && \
+    make install && \
+    mkdir /test && \
+    cp -R test/configs /test/
+
+FROM debian:buster-slim
+
+RUN apt-get update && apt-get install -y uuid libjson-c3 kmod udev
+
+COPY --from=builder /usr/lib64/libaccel-config.so.1.0.0 /lib/x86_64-linux-gnu/
+RUN ldconfig
+
+COPY --from=builder /usr/bin/accel-config /usr/bin/
+COPY --from=builder /test /test
+
+ENTRYPOINT /bin/mount -o remount,rw /sys && /usr/bin/accel-config load-config -c /test/configs/2g2q_user_1.conf && /usr/bin/accel-config enable-device dsa0 && /usr/bin/accel-config enable-wq dsa0/wq0.0 && /usr/bin/accel-config list
```

Comment:
let's try to revisit this `ENTRYPOINT` and by default enable *all* available DSA devices

Reply:
Can we use volume mounts instead of remounting /sys ?

Reply:
@bart0sh 
> Can we use volume mounts instead of remounting /sys ?

If I am not missing anything, we do:

https://github.com/intel/intel-device-plugins-for-kubernetes/blob/9545146ae96d01100085d7ad74d6553858492f57/deployments/dsa_plugin/overlays/dsa_initcontainer/dsa_initcontainer.yaml#L1-L20

I am not quite sure where is the problem, need to dig into this.